app/uk/gov/hmrc/tai/service/UpdateNextYearsIncomeService.scala

```diff
@@ -71,9 +77,9 @@ class UpdateNextYearsIncomeService {
     }
   }
 
-  def setNewAmount(newValue: Int, employmentId: Int, nino: Nino)(implicit hc: HeaderCarrier): Future[UpdateNextYearsIncomeCacheModel] = {
+  def setNewAmount(newValue: String, employmentId: Int, nino: Nino)(implicit hc: HeaderCarrier): Future[UpdateNextYearsIncomeCacheModel] = {
     get(employmentId, nino) map { model =>
-      val updatedValue = model.copy(newValue = Some(newValue))
+      val updatedValue = model.copy(newValue = Some(newValue.toInt))
```

Comment:
Good question, this was the discussion with John about it just being a string. But we will be using it to compare against a current value so it may need to be a number